boards/arm/holyiot_yj16019/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Kconfig - Holyiot YJ-16019 board configuration
+#
+# Copyright (c) 2019 Henrik Brix Andersen <henrik@brixandersen.dk>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_HOLYIOT_YJ16019
```

Comment:
Are you able to remove this empty if/endif block?

Reply:
Fixed, thanks.